content/next/index.md

```diff
@@ -71,39 +73,49 @@ docs: correct spelling of CHANGELOG
 feat(lang): add polish language
 ```
 
-### Commit message for a fix using an (optional) issue number.
+### Commit message with multi-paragraph body and multiple footers
 ```
 fix: correct minor typos in code
 
-see the issue for details on the typos fixed
+see the issue for details
 
-closes issue #12
+on typos fixed.
+
+Reviewed-by: Z
+Refs: #133
 ```
 
 ## Specification
 
 The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
 
 1. Commits MUST be prefixed with a type, which consists of a noun, `feat`, `fix`, etc., followed
-  by an OPTIONAL scope, and a REQUIRED terminal colon and space.
+  by the OPTIONAL scope, OPTIONAL `!`, and REQUIRED terminal colon and space.
 1. The type `feat` MUST be used when a commit adds a new feature to your application or library.
 1. The type `fix` MUST be used when a commit represents a bug fix for your application.
 1. A scope MAY be provided after a type. A scope MUST consist of a noun describing a
   section of the codebase surrounded by parenthesis, e.g., `fix(parser):`
-1. A description MUST immediately follow the space after the type/scope prefix.
+1. A description MUST immediately follow the colon and space after the type/scope prefix.
 The description is a short summary of the code changes, e.g., _fix: array parsing issue when multiple spaces were contained in string._
 1. A longer commit body MAY be provided after the short description, providing additional contextual information about the code changes. The body MUST begin one blank line after the description.
-1. A footer of one or more lines MAY be provided one blank line after the body. The footer MUST contain meta-information
-about the commit, e.g., related pull-requests, reviewers, breaking changes, with one piece of meta-information
-per-line.
-1. Breaking changes MUST be indicated in the type/scope prefix of a commit, or at the beginning of its optional body or footer section.
-1. If included in the body or footer, a breaking change MUST consist of the uppercase text BREAKING CHANGE, followed by a colon, space, and description, e.g.,
+1. A commit body is free-form and MAY consist of any number of newline separated paragraphs.
+1. One or more footer lines MAY be provided one blank line after the body. Each footer MUST
+ follow the [git trailer convention](https://git-scm.com/docs/git-interpret-trailers): consisting of
+ a word token, a colon and space, followed by a string value.
```

Comment:
I'd still like to see us support "pound-sign and a space" as well. This automatically makes `Fixes #12` a footer as well, which fits well with the common pattern people are already doing.

This also fits with the trailer specification itself, in that it actually allows _any_ separator, as long as you configure Git to detect that separator. I'd say for simplicity, this spec should accept either `:<space>` or `<space>#`.